eng/pipelines/dotnet-framework-wcf.yml

```diff
@@ -6,8 +6,8 @@ variables:
 - template: variables/common.yml
 - name: manifest
   value: manifest.wcf.json
-- name: testCategory
-  value: WCF
+- name: repoTestArgs 
+  value: "--filter -Category WCF"
```

Comment:
I wouldn't expect `--filter` to be specified here.  That could be encapsulated within RunTests.